### ISSUE TITLE
Render keyboard with just letters (no numbers)

### ISSUE THREAD

**Issue by alvarosilvac** [WiaczeslawP/react-screen-keyboard#30]
Option where you can render a keyboard of just letters, not including numbers. Easier if you want to prevent the user to add numbers, example, input a name.

**Comment by alvelig** [comment]
I agree, I think that Layout (and customizing it) should include numbers and backspace.
